nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/custom/AdjustContrast.java

```diff
@@ -14,11 +15,11 @@
 
     public AdjustContrast() {super();}
 
-    public AdjustContrast(INDArray in, double factor, INDArray out) {
+    public AdjustContrast(@NonNull INDArray in, double factor, @NonNull INDArray out) {
```

Comment:
Adding `@NonNull` is good in general
but output arrays can be null (inputs must be non-null, however).
Null output means "automatically allocate an output array during execution"
So keep NonNull on in, and remove it from out

Reply:
Done.